Cargo.lock

```diff
@@ -1305,10 +1305,23 @@ checksum = "f6ed9c8b2d17acb8110c46f1da5bf4a696d745e1474a16db0cd2b49cd0249bf2"
 dependencies = [
  "core2",
  "multibase",
- "multihash",
+ "multihash 0.16.3",
  "serde",
  "serde_bytes",
- "unsigned-varint",
+ "unsigned-varint 0.7.1",
+]
+
+[[package]]
+name = "cid"
```

Comment:
@dapplion After thinking about it, I think it would be better if you could relax `cid` version in `rs-car` by using `cid = ">=0.8, <0.11"` in `Cargo.toml`, that way it will automatically resolve to the same version forest uses

Reply:
Looked into it and `libipld-core` depends on cid 0.10.0. That library contains a cbor decoder which may return Cid data structures from the car stream header. libipld-core v0.14.0.

The API of libipld-core v0.14.0 and v0.16.0 (latest that requires cid v0.10.0) is incompatible. The same source can work for both versions.

So I don't see an easy way to support the relaxed range you suggest

Reply:
That makes sense. Thanks for investigating